### PR TITLE
allowed configurable cloud services

### PR DESCRIPTION
**Pull request by filipowm**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -99,7 +99,7 @@ resource "google_compute_network" "gitlab" {
   depends_on              = ["google_project_service.compute"]
 }
 
-resource "google_compute_subnetwork" "us-central" {
+resource "google_compute_subnetwork" "gitlab_subnetwork" {
```
Comment (kcatro) [inline main.tf]: We'll need to rename our resource by hand before running apply next time :D terraform state mv should do the trick I think

File: variables.tf [hunk 2]
```diff
@@ -24,10 +24,51 @@ variable "certmanager_email" {
 
 variable "gitlab_db_password" {
   description = "Password for the GitLab Postgres user"
-  default     = ""
+  default = ""
```
Comment (kcatro) [inline variables.tf]: '=' alignment is proper code style https://www.terraform.io/docs/configuration/style.html

File: main.tf (hunk 1 above)
Comment (filipowm) [reply main.tf]: I reverted for now

File: variables.tf (hunk 2 above)
Comment (filipowm) [reply variables.tf]: done

**Pull request merged**